tests/test_backend.py

```diff
@@ -1,102 +1,102 @@
+from unittest import TestCase
 from unittest.mock import Mock, patch
 
 import django
 from django.core.cache import InvalidCacheBackendError
-from nose.tools import eq_, raises
+from nose.tools import eq_
```

Comment:
`nose` is a testing library that the project used in the past, but they took it out in favor of pytest. There should be no imports from it.